test/controllers/roles_controller_test.rb

```diff
@@ -123,5 +123,15 @@ class RolesControllerTest < ActionController::TestCase
       assert_not_nil cloned_role
       assert_equal @role.permissions, cloned_role.permissions
     end
+
+    test "cloned roles should have builtin value 0 instead of `false`" do
+      params = { :role => {:name => 'clonedrole'},
+                 :original_role_id => @role.id,
+                 :cloned_role => true }
+      post :create, params, set_session_user
+
+      cloned_role = Role.find_by_name('clonedrole')
+      assert_equal cloned_role.builtin, 0
```

Comment:
Might as well put this assertion into the existing cloning test above? It'll save execution time.